eos/constraint_TEST.cc

```diff
@@ -262,7 +267,9 @@ class ConstraintDeserializationTest :
                     "kinematics: {s_min: 15, s_max: 20, s_max: 13.5}\n"
```

Comment:
Why not replace also here s_max -> q2_max and s_min -> q2_min ?!

Reply:
In the above, it was needed since I briefly had to create the constraint to extract the values.
So I didn't even see this one. Will fix that now.